processor/src/chiplets/mod.rs

```diff
@@ -134,6 +134,10 @@ impl Chiplets {
         self.kernel_rom_start() + self.kernel_rom.trace_len()
     }
 
+    pub const fn kernel(&self) -> &Kernel {
```

Comment:
nit: could we add a short doc comment to this function.

Reply:
Done!